config.toml

```diff
@@ -23,17 +25,19 @@ themesDir = "node_modules/"
   header_wrapper_class = "header-default-bg-img"
   js = "js/solstice.js"
   featured_content_publish_target = "openhwgroup"
-# call_for_action_text = ""
-# call_for_action_url = ""
-# call_for_action_icon = ""
+  call_for_action_text = "Download"
```

Comment:
I've asked on the issue ref what the URL should be. For now, it's stubbed.

Reply:
@shanda-eclipse Are we going to use the CTA button in the navbar?

Reply:
Let's hide the button for the initial launch.